jax/_src/xla_bridge.py

```diff
@@ -787,12 +787,15 @@ def _discover_and_register_pjrt_plugins():
   _alias_to_platforms.setdefault(_alias, []).append(_platform)
 
 
+def known_platforms() -> set[str]:
+  return set(_backend_factories.keys()) | set(_platform_aliases.values())
```

Comment:
Not that I changed `_platform_aliases.keys()` to `_platform_aliases.values()`. `_platform_aliases` is currently:
```python
_platform_aliases = {
  "cuda": "gpu",
  "rocm": "gpu",
}
```